radar/dev/vertx.yaml

```diff
@@ -1,27 +1,33 @@
 name: Vert.x
-logo: https://vertx.io/assets/logo-sm.png
+logo: https://upload.wikimedia.org/wikipedia/commons/3/3d/Vertx_logo.png
 blip:
   - date: 2018-03-20
     ring: TRIAL
+  - date: 2021-03-04
+    ring: ADOPT
 description: |
   Eclipse Vert.x is a polyglot event-driven application framework that runs on the
   Java Virtual Machine.
 
   Vert.x contains several different components designed to make it easier for
-  you to write compelling reactive applications in a range of different languages.
+  you to write reactive applications in a range of different languages.
   It is a compelling approach to writing microservices!
```

Comment:
Can you add a reason why it is compelling? I do not know the tool enough to get it from this context. :)